### DISCUSSION TITLE
请问手机上新建的CloudflareST文件夹在哪能找到

### DISCUSSION THREAD

**Discussion by GeorgeRudd** [XIU2/CloudflareSpeedTest#343] [讨论]
mkdir CloudflareST之后在手机上用文件管理器找不到

**Comment by XIU2** [comment]
如果你看的是我写的 #61 教程，那么你是不可能在手机的文件管理器里找到这些文件的。

因为文件都在虚拟机中，即 Termux 是在你手机上创建了一个虚拟的 Linux 微型/精简系统。
因此你手机的文件管理器是无法看到虚拟机内文件的。

所有操作都需要在 Termux（即虚拟机）中进行。

**Comment by GeorgeRudd** [comment]
> 如果你看的是我写的那个教程，那么你是不可能再手机文件管理器里找到的。 因为文件都在虚拟机中。

原来如此 明白了 那可以在手机存储里运行吗 我刚刚 termux-setup-storage 挂载手机存储后 创建storage/shared/termux/CloudflareST 运行脚本提示permission denied

**Comment by XIU2** [comment]
没有权限就赋予权限呗。

不过我对你说的这些操作不熟，那个教程我写完就没在手机上用过了（没这个需求）。。。

**Comment by GeorgeRudd** [comment]
> 没有权限就赋予权限呗。
> 
> 不过我对你说的这些操作不熟，那个教程我写完就没在手机上用过了（没这个需求）。。。

谢谢

**Comment by FireBul** [comment]
[PING][ACTION_REQUIRED]
checked_at_utc: 2026-02-21T22:32:44Z
조건 충족: 60분+ 응답 태그([WORK_REQUEST]/[PROGRESS]/[CRQ_EVIDENCE]) 부재.
요청: 해당 스레드에 WORK_REQUEST 갱신 또는 PROGRESS/CRQ_EVIDENCE로 상태 업데이트 부탁드립니다.
(중복 방지: 동일 스레드 3시간 내 재핑 생략)
